app.js

```diff
@@ -40,10 +40,26 @@ function start (server) {
 
 function setupLogging (settings) {
   loggers.setupAll({
-    enabled: settings.gateway.logEnabled,
-    level: settings.gateway.logLevel,
-    logToFile: settings.gateway.logToFile,
-    filename: settings.gateway.logFilename
+    enabled: settings
```

Comment:
@ahochsteger I suggest to create a default settings object and use extend instead, much cleaner

Reply:
> @ahochsteger I suggest to create a default settings object and use `object.assign` instead, much cleaner

@robertsLando you're absolutely right - it's done.